### ISSUE TITLE
Loading a file generated by EPPlus with rich text causes errors

### ISSUE THREAD

**Issue by vce-wabbott** [PHPOffice/PhpSpreadsheet#2300]
This is:

```
- [X] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?

Should handle loading valid xlsx files that open in Excel 

### What is the current behavior?

Trying to load a file generated by EPPlus with rich text causes errors

Notice: Undefined offset: 179 in /vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Xlsx.php on line 682

from Xlsx.php

// Read cell!
switch ($cellDataType) {
    case 's':
        if ((string) $c->v != '') {
            $value = $sharedStrings[(int) ($c->v)];   // line 682

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php
	require 'vendor/autoload.php';

	use PhpOffice\PhpSpreadsheet\Spreadsheet;
	use PhpOffice\PhpSpreadsheet\Reader\Xlsx;

	$reader = new Xlsx();
	$spreadsheet = $reader->load('Portmap-P1182421-93171.xlsx'); // file attached
```

### Which versions of PhpSpreadsheet and PHP are affected?
PhpSpreadsheet 1.18 and PHP 7.4.11

[Portmap-P1182421-93171.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/7182838/Portmap-P1182421-93171.xlsx)

**Comment by vce-wabbott** [comment]
In my old PHPExcel code this workaround gets around the issue against the same file generated by EPPlus with rich text

https://github.com/PHPOffice/PHPExcel/issues/1187

```
class PHPExcel_Reader_Excel2007_NonStandardNamespacesWorkaround extends PHPExcel_Reader_Excel2007
{
    public function securityScan($xml)
    {
        $xml = parent::securityScan($xml);
        return str_replace(
            [
                '<x:',
                '</x:',
                /*':x=',*/
                '<d:',
                '</d:'
                /*, ':d='*/
            ],
            [
                '<',
                '</',
                /*'=',*/
                '<',
                '</'
                /*, '='*/
            ],
            $xml
        );
    }
}
```

**Comment by oleibman** [comment]
The first phase of having PhpSpreadsheet handle non-standard namespaces has been implemented in master. If you test against master rather than the release version, your failing script will run to a successful conclusion. The support may be incomplete (e.g. cells may not be styled correctly), but all of your data should be available.

**Comment by oleibman** [comment]
Fixed in [1.19.0](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/1.19.0)
